### ISSUE TITLE
site should follow review process too

### ISSUE THREAD

**Issue by 9il** [libmir/mir.rocks#1]

**Comment by wilzbach** [comment]
yes please :)
I just wanted to push an initial version. how do you want to proceed - should I rebase with an empty repo?

**Comment by 9il** [comment]
> should I rebase with an empty repo?

Yes please. Also, i think we need something with more complicated menu like readthedocs (btw, why not to use sphynx? )

**Comment by 9il** [comment]
oh, see, current system is much more simple comparing with Sphynx. reStructuredText allows to import text files. So we are able to check that example are up to date even with Sphynx

**Comment by wilzbach** [comment]
done. I will have a look at Spynx etc - actually I thought about using ddoc again. 
The main feature that we need is that we can verify the correctness of the examples ;-)
